src/main/java/duke/helpers/Ui.java

```diff
@@ -0,0 +1,50 @@
+package duke.helpers;
+
+import java.util.Scanner;
+
+/**
+ * Reads user inputs
+ */
+public class Ui {
+
+    private static Scanner sc;
+
+    /**
+     * Constructs a Ui object and initializes a Scanner object to read user inputs.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Displays a welcome message for the user upon running a Duke object
+     */
+    public void welcome() {
+        System.out.println("Hello! I'm duke.Duke\n" + "What can I do for you?");
+    }
+
+    /**
+     * Reads the next line of user input using a Scanner object
+     *
+     * @return the next line of user input
+     */
+    public static String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Closes the Scanner object
+     */
+    public static void closeScanner() {
```

Comment:
Maybe it will be good to have an exit message implemented here. Not sure if there's any indication of closing the application as I can find it.